lib/src/outputs/advanced_file_output.dart

```diff
@@ -144,5 +149,6 @@ class AdvancedFileOutput extends LogOutput {
     _bufferWriteTimer?.cancel();
     _targetFileUpdater?.cancel();
     await _closeCurrentFile();
+    _buffer.clear();
```

Comment:
This just clears it (deleting the non-flushed data), I meant writing (and clearing) it before ultimately closing the file:
```dart
_bufferWriteTimer?.cancel();
_targetFileUpdater?.cancel();
try {
  _writeOutBuffer();
} catch (e, s) {
  print("Failed to finally flush buffer: $e");
  print(s);
}
await _closeCurrentFile();
```

Reply:
Okay, changed it, and I'll also add a test to send output events and then quickly close the logger.